src/material.cpp

```diff
@@ -503,7 +503,12 @@ std::string pyne::Material::openmc(std::string frac_type) {
 
   // add name if specified
   if (temp_mat.metadata.isMember("mat_name")) {
-    oss << "name=" << new_quote << temp_mat.metadata["mat_name"].asString() << end_quote;
+    std::cout << "Usage of mat_name ad a Metadata is deprecated, please use \"name\" instead." <<std::endl;
```

Comment:
I don't think we need this output and update of the value. I simply wasn't using the right value in the first place and I'm hoping that this capability hasn't been around for long enough to cause a backwards compatibility problem.

Additionally, what if the metadata has both a `name` and `mat_name`? We'd overwrite the exisitng, and possibly preferred, infromation.

Reply:
I removed the search for `mat_name` as advised.
thx for the review !